packages/frontend-api/src/Resources/config/graphql-types/ProductFilter/ProductFilterDecorator.types.yaml

```diff
@@ -0,0 +1,18 @@
+ProductFilterDecorator:
+    type: input-object
+    decorator: true
+    config:
+        description: "Represents a product filter"
+        fields:
+            minimalPrice:
+                type: "Money"
+            maximalPrice:
+                type: "Money"
+            brands:
+                type: "[Uuid]"
+            flags:
+                type: "[Uuid]"
```

Comment:
I think it should be "[Uuid!]" so values are not mandatory (if it's desired behavior), but each element of the array has to be Uuid and cannot be null (empty array is still possible) https://graphql.org/learn/schema/#lists-and-non-null

Reply:
190f141abae92afe3f2fc09f1f542853f4c79247